.github/workflows/build.yml

```diff
@@ -412,7 +412,10 @@ jobs:
           cp liquibase-cdi/target/liquibase-cdi-0-SNAPSHOT.jar artifacts
           cp liquibase-cdi/target/liquibase-cdi-0-SNAPSHOT-sources.jar artifacts
           cp liquibase-cdi/target/liquibase-cdi-0-SNAPSHOT-javadoc.jar artifacts
-          
+
+          cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT.jar artifacts
+          cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT-sources.jar artifacts
+          cp liquibase-cdi-jakarta/target/liquibase-cdi-jakarta-0-SNAPSHOT-javadoc.jar artifacts
```

Comment:
Hello @XDelphiGrl - the copy commands are here , but as we execute the scripts from master they will only run when this pr is merged.

Reply:
Thank you for the explanation, @filipelautert. I'll go ahead and do my code review and we can get this merged. I'll check that the build produces the correctly named artifacts.